src/viam/sdk/rpc/server.cpp

```diff
@@ -55,5 +55,9 @@ void Server::shutdown() {
     }
 }
 
+std::shared_ptr<grpc::Channel> Server::grpc_in_process_channel(const grpc::ChannelArguments& args) {
+    return server_->InProcessChannel(args);
+}
```

Comment:
`client_to_mock_pipeline` needs access to an in process channel, so I exposed one on the `sdk::Server::server_` with this method.